src/seedu/addressbook/storage/jaxb/AdaptedPerson.java

```diff
@@ -60,7 +60,7 @@ public AdaptedPerson(ReadOnlyPerson source) {
 
         address = new AdaptedContactDetail();
         address.isPrivate = source.getAddress().isPrivate();
-        address.value = source.getAddress().value;
+        address.value = source.getAddress().toString();
```

Comment:
Ok to update the unit tests. However, you should also add tests at the system level, i.e. the Actual.txt and the Input.txt.